.ci/Jenkinsfile

```diff
@@ -103,3 +196,63 @@ pipeline {
     }
   }
 }
+
+def notifyStatus(def args = [:]) {
+  def releaseManagerFile = args.get('file', '')
+  def analyse = args.get('analyse', false)
+  def subject = args.get('subject', '')
+  def body = args.get('body', '')
+  releaseManagerNotification(file: releaseManagerFile,
+                             analyse: analyse,
+                             slackChannel: "${env.SLACK_CHANNEL}",
+                             slackColor: 'danger',
+                             slackCredentialsId: 'jenkins-slack-integration-token',
+                             to: "${env.NOTIFY_TO}",
+                             subject: subject,
+                             body: "Build: (<${env.RUN_DISPLAY_URL}|here>).\n ${body}")
+}
+
+def publishArtifacts(def args = [:]) {
+  // Copy those files to another location with the sha commit to test them afterward.
+  googleStorageUpload(bucket: getBucketLocation(args.type),
+    credentialsId: "${JOB_GCS_CREDENTIALS}",
+    pathPrefix: "${BASE_DIR}/build/binaries/",
+    pattern: "${BASE_DIR}/build/binaries/**/*",
+    sharedPublicly: true,
+    showInline: true)
+}
+
+def runReleaseManager(def args = [:]) {
+  deleteDir()
+  unstash 'source'
+  googleStorageDownload(bucketUri: "${getBucketLocation(args.type)}/*",
+                        credentialsId: "${JOB_GCS_CREDENTIALS}",
+                        localDirectory: "${BASE_DIR}/build/binaries",
+                        pathPrefix: getBucketPathPrefix(args.type))
+  dir("${BASE_DIR}") {
+    releaseManager(project: 'elastic-agent-shipper',
```

Comment:
This is the project to be added to the DRA in the Unified Release process, so once it's available then we could test this particular PR (for such we might need to remove some when statements)